pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -571,6 +587,12 @@ func (r *ReconcileClusterDeployment) resolveInstallerImage(cd *hivev1.ClusterDep
 		err = r.Create(context.TODO(), job)
 		if err != nil {
 			jobLog.WithError(err).Error("error creating job")
+		} else {
+			// kickstartDuration calculates the delay between creation of cd and start of imageset job
+			kickstartDuration := time.Since(cd.CreationTimestamp.Time)
+			cdLog.Infof("KICKSTART:%v", kickstartDuration.Seconds())
```

Comment:
Drop this line please.

Reply:
So this is showing over 600 despite it starting immediately? hmmm. I can't see how that would happen...